alphapept/io.py

```diff
@@ -331,6 +331,8 @@ def load_mzml_data(
     mono_mzs_list = []
     charge_list = []
 
+    vendor = "Unknown"
```

Comment:
There are a few redundant codes between load_thermo_raw and load_mzml_data. In the future, we could use a base class to encapsulate these codes.